[Bug]: Multiple manifests in jquery package

### Is there an existing issue for this?

- [x] I have searched the existing issues

### What happened?

Can't install DNN due to having multiple manifests in the jQuery package.

### Steps to reproduce?

1. Download latest CI build
2. Try to install DNN
3. Notice error in jQuery package preventing the install from completing

### Current Behavior

Can't install DNN

### Expected Behavior

Should install

### Relevant log output

```shell
04/21/2026 17:37:29 [ERROR] DotNetNuke.Services.Upgrade.Upgrade Found multiple .dnn files in the zip file: jQuery.dnn and jQuery.Fileupload.dnn
04/21/2026 17:37:29 [ERROR] DotNetNuke.Services.Upgrade.Upgrade Found multiple .dnn files in the zip file: jQuery.dnn and jQuery.iframe-transport.dnn
04/21/2026 17:37:29 [ERROR] DotNetNuke.Services.Upgrade.Upgrade Found multiple .dnn files in the zip file: jQuery.dnn and jQueryMigrate.dnn
04/21/2026 17:37:29 [ERROR] DotNetNuke.Services.Upgrade.Upgrade Found multiple .dnn files in the zip file: jQuery.dnn and jQueryUI.dnn
```

### Anything else?

The package has subfolders in it with related packages and manifests and should not.

### Affected Versions

develop build (unreleased)

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/Build/Tasks/OtherPackages.cs
+++ b/Build/Tasks/OtherPackages.cs
@@ -46,10 +46,10 @@ namespace DotNetNuke.Build.Tasks
         {
             var srcFolder = context.RootDir + context.Directory(package.Folder);
             var files = package.Excludes.Length == 0
-                            ? context.GetFiles($"{srcFolder}**/*")
+                            ? context.GetFiles($"{srcFolder}/**/*")
                             : context.GetFilesByPatterns(srcFolder, IncludeAll, package.Excludes);
             var version = "00.00.00";
-            foreach (var dnn in context.GetFiles($"{srcFolder}**/*.dnn"))
+            foreach (var dnn in context.GetFiles($"{srcFolder}/**/*.dnn"))
             {
                 version = context.XmlPeek(dnn, "dotnetnuke/packages/package/@version");
             }

